External/FEXCore/Source/Interface/Config/Config.cpp

```diff
@@ -719,10 +722,10 @@ namespace JSON {
 #define ENVLOADER
 #include <FEXCore/Config/ConfigOptions.inl>
 
-      EnvMap[Key]=Value;
+      EnvMap[Key] = Value;
     }
 
-    std::function GetVar = [=](const std::string_view id)  -> std::optional<std::string_view> {
+    std::function GetVar = [](EnvMapType &EnvMap, const std::string_view id)  -> std::optional<std::string_view> {
```

Comment:
This doesn't need to be a `std::function` anymore:
```suggestion
    auto GetVar = [](EnvMapType &EnvMap, const std::string_view id)  -> std::optional<std::string_view> {
```

Reply:
Resolved.